browser-test/src/applicant/questions/file.test.ts

```diff
@@ -485,52 +493,55 @@ test.describe('file upload applicant flow', () => {
     }) => {
       await applicantQuestions.applyProgram(programName)
 
-      await applicantQuestions.answerFileUploadQuestion('some file', 'file.txt')
-
-      await applicantQuestions.answerFileUploadQuestion(
-        'some file 2',
-        'file2.txt',
+      await applicantQuestions.answerFileUploadQuestionFromAssets(
+        'file-upload.png',
+      )
+      await applicantQuestions.answerFileUploadQuestionFromAssets(
+        'file-upload-second.png',
       )
 
-      await applicantFileQuestion.removeFileUpload('file.txt')
+      await applicantFileQuestion.removeFileUpload('file-upload.png')
 
-      await applicantFileQuestion.expectFileNameCount('file.txt', 0)
+      await applicantFileQuestion.expectFileNameCount('file-upload.png', 0)
 
-      await applicantFileQuestion.removeFileUpload('file2.txt')
+      await applicantFileQuestion.removeFileUpload('file-upload-second.png')
 
-      await applicantFileQuestion.expectFileNameCount('file2.txt', 0)
+      await applicantFileQuestion.expectFileNameCount(
+        'file-upload-second.png',
+        0,
+      )
     })
 
-    test('too large file error', async ({
-      page,
-      applicantQuestions,
-      applicantFileQuestion,
-    }) => {
-      await applicantQuestions.applyProgram(programName)
+    // TODO remove ".fixme" once https://github.com/civiform/civiform/issues/8143 is fixed
+    test.fixme(
+      'too large file error',
```

Comment:
**question (non-blocking):** What will happen if the file is too large and it breaks the CSP?  Will there be a CSP error in the console, but the user will still see an alert?

Reply:
Yep. Although I honestly haven't tested that exact case. But for all CSP violation cases, things continue to work just fine, the file upload widget just fails to show a preview image and you get an error in the console.